entities/library.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative '../autoload'
+
+# Creates object which represents Library
+# Helps manage the basic needs of Library
+class Library
+  include Statistics
+  include Database
+
+  attr_accessor :authors, :readers, :books, :orders
+
+  def initialize(authors = [], books = [], readers = [], orders = [])
```

Comment:
Let's use keyword arguments for better readability:
`def initialize(authors: [], books: [], readers: [], orders: [])`